mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-parties-7.2.0.plantuml

```diff
@@ -74,149 +75,133 @@ end box
             deactivate CENTRALSERVICE_API
 
-            ALS_CENTRALSERVICE_PARTICIPANT_DAO -> ALS_PARTICIPANT_HANDLER: Return FSPIOP-Source participant information
+            ALS_CENTRALSERVICE_PARTICIPANT_DAO --> ALS_PARTICIPANT_HANDLER: Return FSPIOP-Source participant information
 
             deactivate ALS_CENTRALSERVICE_PARTICIPANT_DAO
 
-            ALS_PARTICIPANT_HANDLER -> ALS_PARTICIPANT_HANDLER: Validate FSPIOP-Source participant. \n<color #FF0000><b>Error code:</b> 320x</color>
+            ALS_PARTICIPANT_HANDLER -> ALS_PARTICIPANT_HANDLER: Validate FSPIOP-Source participant\n<color #red><b>Error code:</b> 320x</color>
         end group
-
-        '********************* Validate Participant - END ************************
         |||
 
         '********************* Request Oracle Participant Information - START ************************
 
-        ALS_PARTICIPANT_HANDLER <-> ORACLE_API: Get Participant Information for PayeeFSP.  \n<color #FF0000><b>Error code:</b> 200x, 310x, 320x</color>
-        ref over ALS_PARTICIPANT_HANDLER
+        ALS_PARTICIPANT_HANDLER <-> ORACLE_API: Get Participant Information for PayeeFSP\n<color #red><b>Error code:</b> 200x, 310x, 320x</color>
+        ref over ALS_PARTICIPANT_HANDLER, ORACLE_API
             GET Participants - [[https://docs.mojaloop.live/mojaloop-technical-overview/account-lookup-service/als-get-participants.html Request Participant Information from Oracle Sequence]]
+            |||
         end ref
 
         '********************* Request Oracle Participant Information - END ************************
         |||
         '********************* Request Parties Information - START ************************
 
-        ALS_PARTICIPANT_HANDLER -> ALS_PARTIES_FSP_DAO: Request Parties information from FSP.\n<color #FF0000><b>Error code:</b> 200x</color>
+        ALS_PARTICIPANT_HANDLER -> ALS_PARTIES_FSP_DAO: Request Parties information from FSP.\n<color #red><b>Error code:</b> 200x</color>
 
         activate ALS_PARTIES_FSP_DAO
-        ALS_PARTIES_FSP_DAO --> PAYEE_FSP: Parties Callback to Destination: GET - /parties/{TYPE}/{ID}?currency={CURRENCY}. \n<color #388724><b>Response code:</b> 202</color> \n<color #FF0000><b>Error code:</b> 200x, 310x, 320x</color>
+        ALS_PARTIES_FSP_DAO ->> PAYEE_FSP: Parties Callback to Destination:\nGET - /parties/{TYPE}/{ID}?currency={CURRENCY}\n<color #388724><b>Response code:</b> 202</color>\n<color #red><b>Error code:</b> 200x, 310x, 320x</color>
         deactivate ALS_PARTIES_FSP_DAO
         activate PAYEE_FSP
 
-        PAYEE_FSP --> ALS_API: Callback with Participant Information: PUT - /parties/{TYPE}/{ID}?currency={CURRENCY}. \n<color #FF0000><b>Error code:</b> 200x, 300x, 310x, 320x</color>
+        PAYEE_FSP ->> ALS_API: Callback with Participant Information:\nPUT - /parties/{TYPE}/{ID}?currency={CURRENCY}\n<color #red><b>Error code:</b> 200x, 300x, 310x, 320x</color>
```

Comment:
We use '-->' to denote an async callback response in our other sequence diagrams do we not?